components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/DefaultTopSitesStorage.kt

```diff
@@ -76,6 +76,16 @@ class DefaultTopSitesStorage(
         }
     }
 
+    override fun editTopSite(topSite: TopSite, title: String, url: String) {
```

Comment:
Let's remove renameTopSite since I don't imagine any consumers of it after this. Can we also call it `updateTopSite` instead of `editTopSite`?

Reply:
Done and done!